### PR TITLE
Get scope dictionary

### PR DESCRIPTION
**Pull request by marianan**
Sample output

Name                           Value                                                                                                                                     
----                           -----                                                                                                                                     
device7                        {global, west, europe}                                                                                                                    
device4                        {global, east, africa}                                                                                                                    
device6                        {global, west, americas}                                                                                                                  
device5                        {global, west, americas}                                                                                                                  
device8                        {global, west, europe}                                                                                                                    
device2                        {global, east, asia}                                                                                                                      
device1                        {global, east, asia}                                                                                                                      
device3                        {global, east, africa}

### CODE REVIEW

File: Get-Scope.ps1 [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        Get-Scope-Array -currentRelativePath $(Split-Path $currentRelativePath)        
+    }
+
+    $scope = $(Split-Path -Leaf $currentRelativePath).Split('.')[0]
+    if(-not $scope.Contains('device')) {
+        $scope
+    }    
+}
+
+Set-Location C:\Work\Git\Fleetbot\deploy
```
Comment (yophilav) [inline Get-Scope.ps1:16]: Not sure if we should use a hard coded path here. Maybe you can try 
`Set-Location "$(git rev-parse --show-toplevel)\deploy" ` ?

File: deploy/global.scope/west.hub/americas.scope/device5.device.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+connection.properties.desired.connecttooverwrite: {
```
Comment (yophilav) [inline deploy/global.scope/west.hub/americas.scope/device5.device.json:1]: Is this change interntional?

Comment (marianan) [reply deploy/global.scope/west.hub/americas.scope/device5.device.json:1]: yes, per morning sync, this is an example of what we would tackle in v2

File: Get-Scope.ps1 (hunk 1 above)
Comment (marianan) [reply Get-Scope.ps1]: Agree, I updated in the next iteration

**Review by yophilav — APPROVED** [review]

**Pull request merged**